src/util.h

```diff
@@ -369,7 +375,7 @@ public slots:
     void keyPressEvent ( QKeyEvent* pEvent )
     {
         // block escape key
-        if ( pEvent->key() != Qt::Key_Escape )
+        if ( ( this->windowFlags() & Qt::Dialog ) != 0 || pEvent->key() != Qt::Key_Escape )
```

Comment:
This seems unrelated at first glance?
If it was really intended to go in, it might make sense to put it into its own commit?

Reply:
Uh...  No idea how that got in there.  Hm... Good spot.  I'd been toying with the idea - it was meant to have been reverted.

Reply:
Strange.  Local diff from here to master doesn't show this as a change.

Reply:
OK, after rebasing I get the change.  It must have been dropped on master and my rebase didn't pick it up for some reason.

Reply:
Hm.  And then it didn't.  I've given in and reverted to just the needed change in the sound header, which seems to make it work.  It would be good to know why the reordering is fine on Linux but not Windows.